seed/data_importer/tests/integration/test_data_import.py

```diff
@@ -449,9 +449,10 @@ def test_map_all_models_xml(self):
         self.assertEqual(ps.count(), 1)
 
         # !! we should have warnings for our file because of the bad measure names !!
-        self.assertNotEqual({}, progress_info.get('file_info', {}))
-        self.assertIn(self.import_file.uploaded_filename, list(progress_info['file_info'].keys()))
-        self.assertNotEqual([], progress_info['file_info'][self.import_file.uploaded_filename].get('warnings', []))
+        self.assertNotEqual({}, progress_info.get('progress_data', {}))
```

Comment:
modify test for `geocode_and_match_buildings_task` new return value

Reply:
noice 👍